tests/system/dub.json

```diff
@@ -3,7 +3,7 @@
     "targetType": "executable",
     "targetPath": "build",
     "dflags": [ "-i", "-preview=in", "-revert=dtorfields" ],
-    "libs-posix": [ "sodium", "sqlite3" ],
+    "libs-posix": [ "sodium", "sqlite3", "openssl" ],
```

Comment:
I don't think you need this, do you ?
The Vibe.d package [depends on `vibe-d:tls` by default](https://github.com/vibe-d/vibe.d/blob/d638b343d6f9ac8da5b18dff61c772d92aa31aa3/dub.sdl#L20) which itself depends on [openssl, including this lib statement](https://github.com/vibe-d/vibe.d/blob/d638b343d6f9ac8da5b18dff61c772d92aa31aa3/tls/dub.sdl#L22-L84).

Reply:
I too expected it to work that way, but it didnt for some reason. I got linking errors without explicit `openssl` in the libraries.

Reply:
I guess we're at that stage where "Because dub" is a good enough explanation for any change.